File: problems/problem03/problem03.c
#include "utility.h"

int* sort(int* a, int sz) {
    /* fill me in */
    return a;
}

int main() {
    int sz;
    int* a = read_ints(stdin, &sz);
    a = sort(a, sz);
    for (int i = 0; i < sz; ++i) {
        printf("%d\n", a[i]);
    }
}

File: problems/problem03/problem03.js
function sort(a) {
    /* fill me in */
    return [];
}


function main() {
    let fs = require('fs');
    let stdin = fs.readFileSync(0).toString();
    let numbers = stdin.trim().split('\n').map(x => parseInt(x));
    let result = sort(numbers);
    for (x of result) {
        console.log(x);
    }
}

main();
